### ISSUE TITLE
5.0.0-beta.14-17a70ee require Angular 6

### ISSUE THREAD

**Issue by xanscale** [angular/flex-layout#714]
and is wrong in my opinion :D
version 5.0.0-beta.14 works fine

npm get automatically the "17a70ee" version. i don't know why

```
npm start

> whereapp-ng@0.0.0 start ******
> ng serve -H=0.0.0.0 -o

** NG Live Development Server is listening on 0.0.0.0:4200, open your browser on http://localhost:4200/ **
 13% building modules 28/29 modules 1 active ...g/node_modules/html-entities/index.jswebpack: wait until bundle finished: /
Date: 2018-04-16T09:15:10.939Z
Hash: c0590aad176e4d6e6997
Time: 8220ms
chunk {inline} inline.bundle.js (inline) 3.85 kB [entry] [rendered]
chunk {main} main.bundle.js (main) 2.91 kB [initial] [rendered]
chunk {polyfills} polyfills.bundle.js (polyfills) 577 bytes [initial] [rendered]
chunk {styles} styles.bundle.js (styles) 190 kB [initial] [rendered]
chunk {vendor} vendor.bundle.js (vendor) 852 kB [initial] [rendered]

ERROR in node_modules/@angular/flex-layout/core/typings/observable-media/observable-media.d.ts(1,22): error TS2305: Module '"/*******/node_modules/rxjs/Rx"' has no exported member 'Subscribable'.


webpack: Failed to compile.
```

**Comment by julianobrasil** [comment]
Change to `6.0.0-beta. 15`. Notice the `6`.

**Comment by xanscale** [comment]
i do not want to use pre-release version of angular.

i think must be:
flex-layout 5.x => angular 5.x
flex-layout 6.x => angular 6.x

actually is
flex-layout 5.x => angular 6.x

**Comment by CaerusKaru** [comment]
The “17a70ee” version was released accidentally and is deprecated on NPM. There will not be another 5.x release in all likelihood, so the fix is to lock your version to 5.0.0-beta.14

**Comment by Teamop** [comment]
@xanscale this is the issue of npm, so you can check [npm v6.0.0-next.0](https://github.com/npm/npm/releases/tag/v6.0.0-next.0) which avoids deprecated versions if possible.

**Comment by frostbytedata** [comment]
I don't even think switching to version `6.0.0-beta.15` resolves this error. It did not seem to fix the error on my local project. Maybe someone else can confirm?

**Comment by Teamop** [comment]
@frostbytedata version `6.0.0-beta.15` means to align release with angular v6 as per angular's strategy, so if you want to use for angular v5, just use `5.0.0-beta.14` right now

**Comment by Mani-RRI** [comment]
working packages versions of **@angular/flex-layout@^2.0.0-beta.9** :

npm install --save @angular/animations@^4.4.5 @angular/cdk@^2.0.0-beta.12 @angular/common@^4.4.5 @angular/compiler@^4.4.5 @angular/core@^4.4.5 @angular/flex-layout@^2.0.0-beta.9 @angular/forms@^4.4.5 @angular/http@^4.4.5 @angular/material@^2.0.0-beta.12 @angular/platform-browser@^4.4.5 @angular/platform-browser-dynamic@^4.4.5 @angular/router@^4.4.5 core-js@^2.5.1 hammerjs@^2.0.8 ngx-restangular@^1.0.11 rxjs@^5.4.3 zone.js@^0.8.18

working packages versions of **@angular/flex-layout@^6.0.0-beta.15** :

"@angular/animations": "^6.0.1",    "@angular/cdk": "^6.0.0",    "@angular/common": "^6.0.0",    "@angular/compiler": "^6.0.0",    "@angular/core": "^6.0.0",    "@angular/flex-layout": "^6.0.0-beta.15",    "@angular/forms": "^6.0.0",    "@angular/http": "^6.0.0",    "@angular/material": "^6.0.1",    "@angular/platform-browser": "^6.0.0",    "@angular/platform-browser-dynamic": "^6.0.0",    "@angular/router": "^6.0.0",    "core-js": "^2.5.4",    "hammerjs": "^2.0.8",    "rxjs": "^6.0.0",    "zone.js": "^0.8.26"  },  "devDependencies": {    "@angular/compiler-cli": "^6.0.0",    "@angular-devkit/build-angular": "~0.6.1",    "typescript": "~2.7.2",    "@angular/cli": "~6.0.1",    "@angular/language-service": "^6.0.0",    "@types/jasmine": "~2.8.6",    "@types/jasminewd2": "~2.0.3",    "@types/node": "~8.9.4",    "codelyzer": "~4.2.1",    "jasmine-core": "~2.99.1",    "jasmine-spec-reporter": "~4.2.1",    "karma": "~1.7.1",    "karma-chrome-launcher": "~2.2.0",    "karma-coverage-istanbul-reporter": "~1.4.2",    "karma-jasmine": "~1.1.1",    "karma-jasmine-html-reporter": "^0.2.2",    "protractor": "~5.3.0",    "ts-node": "~5.0.1",    "tslint": "~5.9.1" 

good luck!

**Comment by CaerusKaru** [comment]
This has been fixed with the release of 5.0.0-beta.15 on NPM
